### DISCUSSION TITLE
ESP32 does not play effects from XLIGTHS in GPIO 2

### DISCUSSION THREAD

**Discussion by santu2898** [forkineye/ESPixelStick#716] [General]
--------- Instructions --------
Please provide answers directly below each section.
--------- Instructions ---------

**ESPixelStick Firmware Version**
ESPixelStick V4 beta5

**Hardware Version**
ESP32 (38 PINES)

**Binary release or compiled yourself?**
Binary

**Operating System (and version)**
windows 10 Home

**Web Browser (and version)**

**Access Point**

**Describe the bug**
I flashed my ESP32 card with ESPixel v4 beta 5, it assigned me an IP and I used it with the Xlights program, but when configuring the outputs on the IP website and later going to the Xlights program, I placed a strip of 50 LEDs to perform a test but it turns out that the LEDs do not have the effect that I choose, it only stays white, I am working with ws2811 LEDs and a 38-pin ESP32 card, could you please help me, I am a new user, thank you in advance

**Comment by MartinMueller2003** [comment]
put the ESP into effects mode (input #2) and set a color. Make sure your lights are that color. If the lights follow the color in the effects setting, then the issue is in xLights. If the color is wrong then you do not have RGB lights and you need to set a different color order.
